### PR TITLE
2020-04-01 update

### PR DESCRIPTION
**Pull request by beckyjackson**
Two new terms:
* `MRO:0002213` - HLA-DQA1*03:03 protein complex	
* `MRO:0002214` - HLA-DQA1*05:05 protein complex

### CODE REVIEW

File: ontology/chain-sequence.tsv [hunk 1]
```diff
@@ -248,7 +248,7 @@ HLA-DQA1*01:04 chain	DQA1*01:04:01:01	IMGT/HLA	HLA00605	MILNKALLLGALALTTMMSPCGGE
...
 HLA-DQA1*03:02 chain	DQA1*03:02	IMGT/HLA	HLA00610	MILNKALMLGALALTTVTSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
-HLA-DQA1*03:03 chain	DQA1*03:03:01	IMGT/HLA	HLA00611	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
+HLA-DQA1*03:03 chain	DQA1*03:03:01:01	IMGT/HLA	HLA00611	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
```
Comment (jamesaoverton) [inline ontology/chain-sequence.tsv:251]: Hmm. Why has this changed?

Comment (beckyjackson) [reply ontology/chain-sequence.tsv:251]: I'm not sure. I ran:
1. `make update-tsv`
2. `make update-iedb`
3. `make update-seqs`

File: mro.owl [hunk 2]
```diff
@@ -1063,8 +1055,6 @@ Some currently missing phenomena that should be considered &quot;about&quot; are
         <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">An evidence type that is based on an assertion by the author of a paper, which is read by a curator.</obo:IAO_0000115>
         <oboInOwl:created_by rdf:datatype="http://www.w3.org/2001/XMLSchema#string">mchibucos</oboInOwl:created_by>
         <oboInOwl:creation_date rdf:datatype="http://www.w3.org/2001/XMLSchema#string">2010-06-21T11:17:21Z</oboInOwl:creation_date>
-        <oboInOwl:hasAlternativeId>ECO:0007010</oboInOwl:hasAlternativeId>
-        <oboInOwl:hasExactSynonym>author inference</oboInOwl:hasExactSynonym>
```
Comment (jamesaoverton) [inline mro.owl:1067]: This is probably because you have a different version of ECO than I do?

File: ontology/chain-sequence.tsv (hunk 1 above)
Comment (jamesaoverton) [reply ontology/chain-sequence.tsv:251]: Ok

File: mro.owl (hunk 2 above)
Comment (jamesaoverton) [reply mro.owl:1067]: I made an issue for this #14

**Comment by rvita** [comment]
I reviewed it on the IMGT/HLA website b/c I was adding the single chain complex & updated it to be correct according to IMGT.

**Pull request merged**